src/frontend/templates/base.html

```diff
@@ -104,16 +104,17 @@
     <div class="container">
       <div class="content has-text-centered">
         <p>
-          <strong>Katago (engine)</strong> by <a href="https://github.com/lightvector">lightvector</a>.
-          <strong>Katago (website)</strong> by <a href="https://github.com/lightvector">lightvector</a> and <a href="https://github.com/tychota">tychota</a>.
-          The source code is licensed <a href="http://opensource.org/licenses/mit-license.php">MIT</a>.
-        </p>
-        <p><a href="/about">Discover more about the project here</a></p>
-        <p>
-          <a class="icon" href="https://github.com/lightvector/KataGo">
-            <i class="fab fa-github"></i>
-          </a>
+          <strong><a href="https://github.com/lightvector/KataGo">Katago engine</a></strong> by <a href="https://github.com/lightvector">lightvector</a>
+          (<a href="https://github.com/lightvector/KataGo/blob/master/LICENSE">engine license</a>).
+          <strong><a href="https://github.com/katago/katago-server">Katago server</a></strong> by <a href="https://github.com/lightvector">lightvector</a> and <a href="https://github.com/tychota">tychota</a>
+          (<a href="http://opensource.org/licenses/mit-license.php">server code license</a>).
         </p>
+        <p><a href="/about">Discover more about the distributed training project here</a></p>
+        <!-- <p> -->
```

Comment:
Oh, I thought the little Github icon link is redundant now that the earlier message has direct links to both the engine and server githubs. I guess I should just delete instead of comment, right?

Reply:
Deleted, and moved the github icons to the new links. Thanks.